ceph.spec.in

```diff
@@ -1175,12 +1175,27 @@ fi
 
 #################################################################################
 %if %{with ocf}
+
 %files resource-agents
 %defattr(0755,root,root,-)
-%dir /usr/lib/ocf
-%dir /usr/lib/ocf/resource.d
-%dir /usr/lib/ocf/resource.d/ceph
-/usr/lib/ocf/resource.d/%{name}/*
+# N.B. %{_libexecdir} is set by unconditional %global to %{_exec_prefix}/lib
```

Comment:
Does `make install` install these files into the directory specified by `./configure --libdir`, or the one specified by `./configure --libexecdir` ?